src/Components/Card/JoinOurTeam/JoinOurTeam.css

```diff
@@ -0,0 +1,36 @@
+.card {
+  padding: 0;
+  width: 1205px;
+  height: 499px;
+  margin: auto;
+  border: none;
+}
+.cardbody img {
+  width: 523px;
```

Comment:
try to use %, make it like 25% so it works on different screen sizes

Reply:
so should i change the width of the card and image to %? and about the rem i will make the px

Reply:
yes, try that something like 25%